### DISCUSSION TITLE
Ability to display where the the app thinks your current location is

### DISCUSSION THREAD

**Discussion by LiminalFish** [breezy-weather/breezy-weather#1155] [Ideas]
### Describe your idea

I have noticed occasionally discrepancies between the data reported by the current location and where I know I am. For example, I know that I am in Austin, TX and I have it added as a location. The info displayed in the Austin tab and the info displayed in the current location tab do not always match up. I believe that allowing the user to see where the app thinks their current location is would significantly the decision making process for what information to use.
![Screenshot_20240623_100218_Breezy Weather](https://github.com/breezy-weather/breezy-weather/assets/43814809/7b9ef254-e53f-4a09-847a-d18e60506c63)

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.4 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Have a look at main sources supporting reverse geocoding: https://github.com/breezy-weather/breezy-weather/blob/main/docs/SOURCES.md
